cartographer_ros/cartographer_ros/node.cc

```diff
@@ -52,6 +52,7 @@ cartographer_ros_msgs::SensorTopics DefaultSensorTopics() {
   topics.point_cloud2_topic = kPointCloud2Topic;
   topics.imu_topic = kImuTopic;
   topics.odometry_topic = kOdometryTopic;
+  topics.fixed_frame_pose_topic = kFixedFramePoseTopic;
   return topics;
```

Comment:
You need to add a SubscribeWithHandler to subscribers_ in Node::LaunchSubscribers

Reply:
Done. Thanks!